src/vochain/vochain.proto

```diff
@@ -96,6 +98,24 @@ message ProofArbo {
         bytes siblings = 2;
 }
 
+// Groth16 zkSNARK proof + public inputs
+message ProofZkSNARK {
+        // Type determines which to circuit corresponds the zkProof
+        enum Type {
```

Comment:
There will be different circuits of the `zk-census-proof` depending on the census size, also there might be more use cases with different circuit designs.
The client & Vochain need a way to unequivocally identify those circuits, in order to user the proper `Proving key`, `Witness calculator` and `Verification key` for each circuit.

The following defines the way how the circuits are identified in the client and also inside the Vochain through a Protobuf enum type. Each `ProofZkSNARK` protobuf packet will have a `Type` identifier that states to which circuit belongs the proof.

Format: `CIRCUITNAME_PARAMETER1_PARAMETER2`

List of current types:
- `ZKCENSUSPROOF_NLEVELS_NMINERS`
    - Circuit name: [`ZKCENSUSPROOF`](https://github.com/vocdoni/zk-franchise-proof-circuit/blob/master/circuits/census.circom)
    - Parameters: `nLevels`, `nMiners`
    - Example:
        - `ZKCENSUSPROOF_100_5`
        - `ZKCENSUSPROOF_1000_20`

At the moment of this PR there is no defined circuit parameters, so the available options in the enum Type is only `UNKNOWN`. But once is defined the circuit parameters for the `zk-census-proof` circuit, we can add them here.

Reply:
According to the docs PR, we should somehow include the two size parameters?

Reply:
The `ProofZkSNARK` is intended to be used for different snark circuits, abstracted from specific parameters of a specific circuit. As for the moment there are no circuits defined, for the moment there is the type `UNKNOWN`.
Once we have defined the specific circuits to be used for `zk-census-proof` protocol (and also for other future circuits), they will be added into this enum following the spec being proposed in the docs PR: https://github.com/vocdoni/docs/pull/34/files#diff-7562f547da8fabde08ab68c3d27b466d13e7328172a3f7717646362e8917682dR212

It will look something like:
```js
enum Type {
        UNKNOWN = 0;
        ZKCENSUSPROOF_100_5 = 1; // for the zk-census-proof circuit with 100 levels & 5 miners
        ZKCENSUSPROOF_1000_11 = 2;
        ANOTHERFUTURECIRCUIT_400 = 3; // a future circuit that has 1 parameter
}
```

Reply:
I don't understand the "Nminers" parameter, can you clarify?

Reply:
The `nMiners` parameter of the template of the circuit: https://github.com/vocdoni/zk-franchise-proof-circuit/blob/master/circuits/census.circom#L66

Reply:
Ah right. This is outdated/legacy, with @adria0 we decided to keep it but its not gonna be used. So I'd not add this parameter to the protobuf enum.

Reply:
Cool, updated the spec (removing the relayer keys related stuff(`nMiners`)): https://github.com/vocdoni/docs/pull/34

Reply:
great :+1: